### PR TITLE
New Strimzi Jekyll Theme

### PR DESCRIPTION
**Pull request by adelasofia**
This PR contains the new Strimzi Jekyll theme. 

Nothing with regards to the site build should have changed. A normal [Jekyll installation](https://jekyllrb.com/docs/) is all that is needed. 

I tried to keep the structure and contents of the directories as similar as the previous Strimzi Jekyll site, but please build the site locally to be sure no important content is missing.

Let me know if there are any issues, I'm always happy to help.

### CODE REVIEW

**Comment by ppatierno** [comment]
@adelasofia thanks! 
I am not a Ruby expert but I have just tried to build the website locally, getting the following error when I run `bundle exec jekyll serve`

```
bundler: failed to load command: jekyll (/usr/local/bin/jekyll)
LoadError: libruby.so.2.5: cannot open shared object file: No such file or directory - /usr/lib64/gems/ruby/ffi-1.9.25/ffi_c.so
  /usr/share/gems/gems/ffi-1.9.25/lib/ffi.rb:6:in `require'
  /usr/share/gems/gems/ffi-1.9.25/lib/ffi.rb:6:in `rescue in <top (required)>'
  /usr/share/gems/gems/ffi-1.9.25/lib/ffi.rb:3:in `<top (required)>'
  /usr/share/gems/gems/rb-inotify-0.9.10/lib/rb-inotify/native.rb:1:in `require'
  /usr/share/gems/gems/rb-inotify-0.9.10/lib/rb-inotify/native.rb:1:in `<top (required)>'
  /usr/share/gems/gems/rb-inotify-0.9.10/lib/rb-inotify.rb:2:in `require'
  /usr/share/gems/gems/rb-inotify-0.9.10/lib/rb-inotify.rb:2:in `<top (required)>'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/linux.rb:30:in `require'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/linux.rb:30:in `_configure'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/base.rb:45:in `block in configure'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/base.rb:40:in `each'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/base.rb:40:in `configure'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/adapter/base.rb:63:in `start'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/backend.rb:28:in `start'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/listener.rb:67:in `block in <class:Listener>'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/fsm.rb:120:in `instance_eval'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/fsm.rb:120:in `call'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/fsm.rb:91:in `transition_with_callbacks!'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/fsm.rb:57:in `transition'
  /usr/share/gems/gems/listen-3.0.8/lib/listen/listener.rb:90:in `start'
  /usr/local/share/gems/gems/jekyll-watch-2.0.0/lib/jekyll/watcher.rb:26:in `watch'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/build.rb:94:in `call'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/build.rb:94:in `watch'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/build.rb:43:in `process'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/serve.rb:93:in `block in start'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/serve.rb:93:in `each'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/serve.rb:93:in `start'
  /usr/share/gems/gems/jekyll-3.8.4/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
  /usr/local/share/gems/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
  /usr/local/share/gems/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
  /usr/local/share/gems/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
  /usr/local/share/gems/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
  /usr/local/share/gems/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
  /usr/share/gems/gems/jekyll-3.8.4/exe/jekyll:15:in `<top (required)>'
  /usr/local/bin/jekyll:23:in `load'
  /usr/local/bin/jekyll:23:in `<top (required)>'
```

Anyway the file it is looking for already exists in `/usr/lib64/gems/ruby/ffi-1.9.25/ffi_c.so`.
I also tried to do a `gem install bundler` and `bundle install` before (without knowing what I was doing :-)). FYI, I am able to build the original web site locally.
Can you help me on this? Thanks!

**Comment by tombentley** [comment]
@adelasofia you'll need to sign off on your commits (`git rebase --signoff` and a force push should do it)

File: _includes/community/community-contributions-band.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<div class="component-wrapper contributions-band">
+  <div class="width-3-12 width-12-12-m contrib-block">
```
Comment (tombentley) [inline _includes/community/community-contributions-band.html:2]: Presumably these are all supposed to be links?

Comment (adelasofia) [reply _includes/community/community-contributions-band.html:2]: No problem. Do these seem like the correct links to you? 

Download Strimzi: https://deploy-preview-130--strimzi.netlify.com/downloads
Follow Strimzi on Twitter: https://twitter.com/strimziio
Submit a bug: https://github.com/strimzi/strimzi-kafka-operator/issues
Submit some code: https://github.com/strimzi/strimzi-kafka-operator/labels/good-start
Join our chatroom: https://slack.cncf.io/
Edit the Documentation: https://deploy-preview-130--strimzi.netlify.com/contributing/guide/
Blog with us: https://github.com/strimzi/strimzi.github.io
Translate into your language: https://deploy-preview-130--strimzi.netlify.com/contributing
Spread the word: https://twitter.com/strimziio

They're just my best guesses :)

Comment (scholzj) [reply _includes/community/community-contributions-band.html:2]: We obviously need the links to not use the `https://deploy-preview-130--strimzi.netlify.com` part, but I guess that is clear. Apart frm that I think these would be a good start.

**Comment by scholzj** [comment]
> "Kafka on Kubernetes in 60 seconds" should be more "Kafka on Kubernetes in 2 minutes" or something like that. @tombentley @scholzj ?

Given the length of the vide, _Kafka on Kubernetes in few minutes_ sounds best to me.

> In the home page, the link at "What is an operator" seems to be wrong. It links to the doc but I remember that the original link was more something like this (https://coreos.com/blog/introducing-operators.html). Is that right @scholzj ? The website should also be opened in a different window to avoid people leaving the Strimzi website.

I think opening it in new window is a bit weird. It sounds like 199X to me. What about staying on our site and linking here? https://deploy-preview-130--strimzi.netlify.com/docs/overview/latest/#overview-components_str (Maybe we can add link to the CoreOS page from the docs)

What I would do is remove the `and why should I use them?` part and leave it only as `What is an operator?` and better align with the other 2 columns next to it.

> in the doc: "When the google search ... " should be with capital "G"

Not a linguist, but I do not think it is necessarily refering to brand but ratehr to _googling_. So maybe it is ok?

> in the blog section, I think that it could be great to have the first part of the blog text in the list as well. It's something that we already have today with current website. The new website just shows the title as far as I can see.

I can see cons and pros. I think it could be a long boring page with the perexes there. We need to get some pictures to add there to make it more lively I think. MAybe until then just list of titles is better?

-----

I think it also dropped the _Kind quickstart_. 

But overall I think maybe we should solve only the really critical things now, merge this and then we can do some smaller things later. For me the _60 seconds_ look stupid with much longer video. The rest can be done also later. The longer we keep this open the more new differences will happen.

**Comment by ppatierno** [comment]
> I can see cons and pros. I think it could be a long boring page with the perexes there. We need to get some pictures to add there to make it more lively I think. MAybe until then just list of titles is better?

Tbh I don't like the raw list, I like to read part of the article which should attract me to read in entirely.

**Comment by adelasofia** [comment]
I have pushed the changes requested by @tombentley for the /community page. Let me know if there are any final decisions on the other suggestions and I'd be happy to make the changes.

**Comment by ppatierno** [comment]
@adelasofia maybe changing the main statement in "Kafka on Kubernetes in few minutes"

**Comment by adelasofia** [comment]
> @adelasofia maybe changing the main statement in "Kafka on Kubernetes in few minutes"

Done 👍

**Comment by scholzj** [comment]
Thanks a lot for the all the work on this @adelasofia

**Pull request merged**